addons/sourcemod/scripting/SprayManager.sp

```diff
@@ -3016,7 +3016,7 @@ void UpdateNSFWInfo(int client)
 		return;
 
 	char sSprayQuery[128];
-	Format(sSprayQuery, sizeof(sSprayQuery), "SELECT * FROM `spraynsfwlist` WHERE `sprayhash` = '%s';", g_sSprayHash[client]);
```

Comment:
Why select 1, i think the callback fonction still needs to access some values in the returned row

Reply:
Since the spray hash is not shared (excepted for original game sprays), their no reason to try get more than one result ?

Reply:
Here:
- https://github.com/srcdslab/sm-plugin-SprayManager/blob/00f0ff9eea618d301b07c8618f4fdde6b68bbda8/addons/sourcemod/scripting/SprayManager.sp#L3099